new/components/basic/layout/BaseHeader.tsx

```diff
@@ -1,19 +1,12 @@
-import { PropsWithChildren } from "react";
-import { StyleSheet, Text } from 'react-native';
+import React, { PropsWithChildren } from "react";
+import { BaseLargeText } from "./BaseText";
 
 
 interface BaseHeaderProps extends PropsWithChildren<{}> {
 }
 
 export default function BaseHeader({ children, ...props }: BaseHeaderProps) {
     return (
-        <Text style={styles.header}>{children}</Text>
+        <BaseLargeText>{children}</BaseLargeText>
```

Comment:
what is the purpose for wrapping BaseLargeText with BaseHeader?
If there is no extra logic added to it